design-proposals/vpro-device.md

```diff
@@ -0,0 +1,196 @@
+# Design Proposal: vPRO/AMT/ISM devices activation
+
+Author(s): Edge Infrastructure Manager Team
+
+Last updated: 05/12/2025
+
+## Abstract
+
+vPRO/Active Management Technology (AMT)/Intel Standard Manageability (ISM) needs to be explicitily activated and
+configured in the devices before to be available in the Edge Orchestrator.
+
+[Open Device Management Toolkit](https://device-management-toolkit.github.io/docs/2.27/GetStarted/overview/) (Open DMT)
+provides an open source stack through which is possible to manage vPRO enabled devices.
+
+![EMF Stack](./images/specs/stack.svg)
+
+This document describes the design proposal for adding the Remote Provisioning Client (RPC) component of DMT stack into
+the EN sw and propose a design on how to manage the device activation during the EN journey.
+
+## Proposal
+
+Remote Provisioning Client (RPC) is launched during the provisioning flow (uOS stage).
+
+vPRO (AMT & ISM) capable Core devices need to have the `rpc-go` utility embedded in the uOS image (the so called
+Tink-EMT image - replacement of HookOS), as this is required to activate AMT. This also requires the `heci` driver to
```

Comment:
Even though we are using tink-EMT we will be enabling vPRO on both Ubuntu and EMT, correct ?

Reply:
yes but we will not control the activation, reconfiguration in the final OS. This will come later (in 3.2) with the help of the Platform Manageability Agent